universal-login-example/src/components/Account.js

```diff
@@ -28,14 +27,15 @@ class Account extends Component {
         <div className="container">
           <ProfileIdentity
             type="identityAccount"
-            identityService={this.props.identityService}/>
+            identityService={this.props.identityService}
+          />
           <hr className="separator" />
-          <ManageDevicesAccordion emitter={this.props.identityService.emitter} />
+          <ManageDevicesAccordion
+            emitter={this.props.identityService.emitter}
+          />
           <hr className="separator" />
           <BackupCodeAccordionView setView={this.setView.bind(this)} />
           <hr className="separator" />
-          <TrustedFriendsAccordionView setView={this.setView.bind(this)} />
-          <hr className="separator" />
```

Comment:
Because it's not in scope, no one is working to implement it and it adds complexity to it. If someone has actual plans to do it, then we could put it back